app/src/main/java/com/hamdan/acceleromatorminiproject/data/ScoreEntity.kt

```diff
@@ -0,0 +1,21 @@
+package com.hamdan.acceleromatorminiproject.data
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.hamdan.acceleromatorminiproject.data.ScoreEntity.Companion.SCORE_TABLE_NAME
+
+@Entity(tableName = SCORE_TABLE_NAME)
+data class ScoreEntity(
+    @ColumnInfo(DATE_ACQUIRED) var dateAcquired: Long?,
+    @ColumnInfo(ELAPSED_TIME) var elapsedTime: Long?,
+) {
+    @PrimaryKey(autoGenerate = true)
+    var id: Int = 0
+
+    companion object {
+        const val SCORE_TABLE_NAME = "scores"
+        const val DATE_ACQUIRED = "date_acquired"
```

Comment:
as a side note, its sometimes worth it to put a differentiator at the front of nameing for sql column names. Lets say theres two tables in a database, legacy and active, if both have date_acquired as a column name the sql won't be able to differentiate unless you specify further or you make the nameing different enough.

Reply:
Considering there's only one table here it's not an issue, but just something to remember for future projects that include the usage of SQL.

Reply:
Good catch, updating these to be safe for multiple tables